m4_sessions/1711-inning/Group_4.md

```diff
@@ -1,9 +1,25 @@
 Group Four
 
-Session Title: Description: Why should students come to this session?
+Session Title: How to write a kicka$$ technical blog post
+
+Description: Why should students come to this session?
+
+  * A technical blog will highlight students programming strengths and will convey what they currently know.
+  * It also serves as good self-marketing (which can lead to a job!)
 
 Session Outcomes: What will students leave with or gain from this session?
 
+  * They will gain the practical skillset to write their first technical article.
+
 Session Outline: Include the flow of the session and the time for each section.
 
+  * WHY - why it's important to write a technical blog post
+  * WHAT - topic of the blog
+  * HOW - how to write a technical blog (provide them a list of guidelines that will allow them to start writing that day)
```

Comment:
This is a great outline.  Will you as facilitators have multiple examples of blog posts? Will you break them into small groups for the writing time?  Will you have a template of how to write the post?  Can you create a slack channel for them and share resources or links to the final article?